addons/stock/wizard/stock_replenishment_info.xml

```diff
@@ -7,13 +7,13 @@
             <form>
                 <field name="orderpoint_id" invisible="1"/>
                 <field name="qty_to_order" invisible="1"/>
-                <div class="row ml8">
-                    <group class="col-6">
-                        <field nolabel="1" name="json_lead_days" widget="lead_days_widget"/>
-                    </group>
-                    <group class="col-6">
-                        <field nolabel="1" name="json_replenishment_history" widget="replenishment_history_widget"/>
-                    </group>
+                <div class="row">
+                    <div class="col-md-6 col-sm-8">
+                        <field name="json_lead_days" widget="lead_days_widget"/>
+                    </div>
+                    <div class="col-md-6 col-sm-4">
+                        <field class="d-block" name="json_replenishment_history" widget="replenishment_history_widget"/>
+                    </div>
```

Comment:
We should instead use the classic structure
```xml
<group>
  <group>
    <field name="json_lead_days" no_label="1" widget="lead_days_widget"/>
  </group>
  <group>
    <field name="json_replenishment_history" no_label="1"  widget="replenishment_history_widget"/>
  </group>
</group>
```
And check in the widget template why it's not adapting correctly

Reply:
concurrent comments :)

use the colspan attribute, see below!